src/ic-certified-assets/src/state_machine.rs

```diff
@@ -564,70 +564,6 @@ impl From<StableState> for State {
     }
 }
 
-fn decode_etag_seq(value: &str) -> Result<Vec<Hash>, String> {
```

Comment:
We're planning to reintroduce this functionality soon.  Can we prefix this with `_` or otherwise disable the warning for this method?  This way no one will have to hunt through git history to find this in the future.

Reply:
Yes, I prefix the function with `_`.